### DISCUSSION TITLE
mysql/basic/202202/1.MySQL%E5%A4%87%E4%BB%BD%E6%81%A2%E5%A4%8D%E6%9C%80%E4%BD%B3%E5%AE%9E%E8%B7%B5

### DISCUSSION THREAD

**Discussion by giscus** [blylei/blylei.github.io#3] [General]
# mysql/basic/202202/1.MySQL%E5%A4%87%E4%BB%BD%E6%81%A2%E5%A4%8D%E6%9C%80%E4%BD%B3%E5%AE%9E%E8%B7%B5

How we rebuilt client-side search, delivering a better user experience while making it faster and smaller at the same time

https://blog.blylei.tech/mysql/basic/202202/1.MySQL%E5%A4%87%E4%BB%BD%E6%81%A2%E5%A4%8D%E6%9C%80%E4%BD%B3%E5%AE%9E%E8%B7%B5.html

**Comment by blylei** [comment]
写的真好
